core/io/resource_uid.cpp

```diff
@@ -35,6 +35,8 @@
 #include "core/io/dir_access.h"
 #include "core/io/file_access.h"
 
+// These constants are off by 1, causing the 'z' and '9' characters never to be used.
+// This cannot be fixed without breaking compatibility; see https://github.com/godotengine/godot/issues/83843
```

Comment:
We usually reference GitHub issues like this (and added a final dot as per out comment style).
```suggestion
// This cannot be fixed without breaking compatibility; see GH-83843.
```